services/movie_session.py

```diff
@@ -42,3 +44,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict[str, Any]]:
+    tickets = MovieSession.objects.filter(movie_session_id=movie_session_id)
+    return [{"row": ticket.row, "seat": ticket.seat} for ticket in tickets]
```

Comment:
The `get_taken_seats` function is attempting to filter `MovieSession` objects by `movie_session_id`, which is incorrect. You should be filtering `Ticket` objects instead, as they contain the `row` and `seat` information. Change `MovieSession.objects.filter(movie_session_id=movie_session_id)` to `Ticket.objects.filter(movie_session_id=movie_session_id)`.